tools/wptrunner/wptrunner/testloader.py

```diff
@@ -51,6 +51,16 @@ def __contains__(self, key):
     def __getitem__(self, key):
         return self._data[key]
 
+def update_include_from_file(file):
```

Comment:
Can we call this `read_include_from_file` instead, since it doesn't actually update anything.

Reply:
sure, updated